src/Layout/AppLayout.jsx

```diff
@@ -9,6 +9,7 @@ import { Icon } from '../Icon'
 
 export const AppLayout = ({
   sidebarContent,
+  sidebarContentCss,
```

Comment:
Can't we do `sidebarCss`? It does seem like the applied CSS is on sidebar iteself, not on content